pyproject.toml

```diff
@@ -1,50 +1,67 @@
 [tool.poetry]
-name = "open-rarity"
-version = "0.5.1-beta"
+authors = ["Dan Meshkov <daniil.meshkov@opensea.io>", "Vicky Gong <vicky.gong@opensea.io>", "Taylor Beever <taylor@quicknode.com>"]
 description = "Open-Rarity library is an open standard that provides an easy, explanable and reproducible computation for NFT rarity"
-authors = ["Dan Meshkov <daniil.meshkov@opensea.io>", "Vicky Gong <vicky.gong@opensea.io>"]
 license = "Apache-2.0"
+name = "open-rarity"
+packages = [
+  {include = "openrarity"},
+  {include = "openrarity/py.typed"},
+]
+version = "1.0.0a1"
 
 readme = "README.md"
 
 classifiers = [
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python",
-    "Intended Audience :: Developers",
-    "License :: OSI Approved :: Apache Software License",
-    "Natural Language :: English",
-    "Typing :: Typed",
-    "Topic :: Software Development :: Libraries",
-    "Development Status :: 4 - Beta",
-    "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: Apache Software License",
+  "Natural Language :: English",
+  "Typing :: Typed",
+  "Topic :: Software Development :: Libraries",
+  "Development Status :: 4 - Beta",
+  "Programming Language :: Python :: 3.10",
 ]
+[tool.poetry.scripts]
+openrarity = "openrarity.cli.main:app"
 
 [tool.poetry.dependencies]
-python = ">=3.10,<3.11"
-pydantic = "^1.9.1"
-numpy = "^1.23.1"
-pandas = "^1.4.3"
-scipy = "^1.9.0"
-requests = "^2.28.1"
+aiolimiter = "^1.0.0"
```

Comment:
i'd like us to keep depdency footprint limited. Let's discuss in chat  what each new dependency brings.